intermine_boot/docker.py

```diff
@@ -65,20 +66,13 @@ def _create_volume_dirs(compose_path):
 def up(options, env):
     compose_path = _get_compose_path(options, env)
 
-    same_conf_exist = False
-    if compose_path.parent.is_dir():
-        if _is_conf_same(env['data_dir'], options):
-            print ('Same configuration exist. Running local compose file...') 
-            same_conf_exist = True
-        else:
-            print ('Configuration change detected. Downloading compose file...')
-            shutil.rmtree(compose_path.parent)
-    
-    if not same_conf_exist:
-        Repo.clone_from(DOCKER_COMPOSE_REPO, compose_path.parent, 
-                    progress=utils.GitProgressPrinter())
+    if (env['data_dir'] / 'docker').is_dir():
```

Comment:
Just want to point out that the intent of this was to delete the volume directories which were inside the `docker` folder, so that the containers could be started with empty volume directories and therefore start building the mine from scratch.

You don't need to fix this now, but we will need to fix it in #25 by having it delete the volume directories directly instead, once this PR gets merged.

Reply:
ok